src/client/aliasviewer.py

```diff
@@ -146,7 +146,7 @@ def name_used_by_others(self, others, original_user=None):
 
         others = [u for u in others if u['name'] != original_user]
         if len(others) == 0 and original_user is None:
-            return 'The name has never been used.'
+            return 'The name has never been used.<br/><br/>(the search is case sensitive)'
```

Comment:
IMO it'd be better to add "(case sensitive)" in line 218 (setLabelText), since this formatting is also used by the player context menu. Added bonus of letting the user know right away that it's case sensitive.

Reply:
the case 'user doesn't exist' will not come up from context menu, so ...
but ok ...not too much text?
![fafclient-chat-alias-search](https://user-images.githubusercontent.com/19195504/34230667-1baedab2-e5d9-11e7-8b57-09cf2cdfb954.jpg)

Reply:
It fits, so I think it's good.